### PR TITLE
fix(ColorPicker): ToolTip of ColorPickerSlider doesn't work properly

### LINKED ISSUE #13868: [WASM] Tooltip's position is not correct.
### Current behavior

When I close the color picker, strange tooltip appears and doesn't go away at the top left of page.
This issue is only occurred when color picker is in the ElevatedView and when removing the ElevatedView from Visual Tree. (I attached example)

image
[UnoApp1-master.zip](https://github.com/unoplatform/uno/files/12765167/UnoApp1-master.zip)

### Expected behavior

_No response_

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by TopProgrammer77**
GitHub Issue (If applicable): closes #13868

## PR Type

What kind of change does this PR introduce?

- Bugfix
### <samp>🤖 Generated by Copilot at 2234ae9</samp>

This pull request enhances the `ColorPicker` and `ColorPickerSlider` controls, fixes a bug with the `Slider` thumb tooltip, and adds a new UI test and a new sample page for the `ColorPicker`. It also adds some null checks and flags to improve the stability and performance of the `ToolTip` and `Slider` controls.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs [hunk 1]
```diff
@@ -172,11 +173,8 @@ private void OnValueChangedEvent(object sender, RangeBaseValueChangedEventArgs a
 			if (m_toolTip is ToolTip toolTip)
 			{
 				toolTip.Content = GetToolTipString();
-
-				// ToolTip doesn't currently provide any way to re-run its placement logic if its placement target moves,
-				// so toggling IsEnabled induces it to do that without incurring any visual glitches.
-				toolTip.IsEnabled = false;
-				toolTip.IsEnabled = true;
```
Comment (Youssef1313) [inline src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs:179]: This code matches WinUI and shouldn't be changed. I believe @jeromelaban was already working on the proper fix.

Comment (TopProgrammer77) [reply src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs:179]: I fixed 2 issues( #13868 and #13865) in this PR. Which issue was @jeromelaban working on?

Comment (TopProgrammer77) [reply src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs:179]: @jeromelaban , @Youssef1313 .
No need to PR for issues #13868 and #13865?

**Comment by DierkDroth** [comment]
@jeromelaban @Youssef1313 Is there any chance to get this PR reviewed? I would be happy to test that fix as those issues manifest themselves in my UNO WASM app.

Thanks in advance

File: src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs:179]: Indeed, we cannot change that code as the original behavior from winui needs to be kept and tooltip needs to get the behavior that ColorPickerSlider is depending on. I worked on this during last week's stream, see https://www.youtube.com/watch?v=8Txc13MMJQk, though I did not get to finish it.

Comment (jeromelaban) [reply src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs:179]: @DierkDroth not at this time, but it can be reused to update this PR.

Comment (DierkDroth) [reply src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs:179]: Ah, thanks @jeromelaban for letting us know.

**Comment by TopProgrammer77** [comment]
@jeromelaban @Youssef1313 
I removed the fix for issue #13865 and this PR is only for the issue #13868.
And I will create new PR for #13865.

**Comment by TopProgrammer77** [comment]
This PR isn't needed now because I can't reproduce the issue on the latest Uno.
Thank you.

**Pull request closed without merging**